applications/filestore-backup/templates/cronjobs.yaml

```
@@ -0,0 +1,97 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: create-backup
+spec:
+  schedule: {{ .Values.tool.backup.schedule | quote }}
+  successfulJobsHistoryLimit: 1
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          restartPolicy: Never
+          automountServiceAccountToken: true
+          {{- with .Values.tolerations }}
+          tolerations:
+{{ toYaml . | indent 12 }}
```

```suggestion
          tolerations:
            {{ toYaml . | nindent 12 }}
```
and similarly everywhere else you have this construct, so that it will match all of our other charts.